defaults/main/drop_lasso.yml

```
@@ -0,0 +1,13 @@
+---
+
+
+pf_drop_lasso_template: default-drop-lasso.j2
+
+# this is IP4 and IPv6 respectively
+# see https://www.spamhaus.org/drop/
+#
+pf_drop_lasso_v4: True
+pf_drop_lasso_v6: True
```

Let's use consistently lowercase boolean across the role.

```suggestion
pf_drop_lasso_v4: true
pf_drop_lasso_v6: true
```